vim_menutrans_helper.py

```diff
@@ -0,0 +1,196 @@
+import re
+import glob
+
+menu_item = {"me", "menu", "noreme", "noremenu", "am",
+             "amenu", "an", "anoremenu", "nme", "nmenu",
+             "nnoreme", "nnoremenu", "ome", "omenu", "onoreme",
+             "onoremenu", "vme", "vmenu", "vnoreme", "vnoremenu",
+             "xme", "xmenu", "xnoreme", "xnoremenu", "sme", "smenu",
+             "snoreme", "snoremenu", "ime", "imenu", "inoreme", "inoremenu",
+             "cme", "cmenu", "cnoreme", "cnoremenu", "tlm", "tlmenu", "tln",
+             "tlnoremenu", "E330", "E327", "E331", "E336", "E333", "E328", "E329",
+             "E337", "E792", "tunmenu", "tu", "popup", "popu"}
+toolbar_item = {"tmenu", "tm"}
+
+# unescape double quotes
+def unescape_double_quotes(origin):
+    origin = origin.replace('\\"', '"')
+    origin = origin.replace("\\'", "'")
+    origin = origin.replace('\\t', '\t')
+    origin = origin.replace('\\\\', '\\')
+    return origin
```

Comment:
Actually you can chain the return values. It won't make it more efficient to run, but can make it easier to read:

```
    return (origin.replace('\\"', '"')
                  .replace("\\'", "'")
                  .replace('\\t', '\t')
                  .replace('\\\\', '\\'))
```